fdbclient/BackupContainer.actor.cpp

```diff
@@ -1180,7 +1180,31 @@ class BackupContainerLocalDirectory : public BackupContainerFileSystem, Referenc
 		// can't be used because backup files are read-only.  Cached mode can only help during restore task retries handled
 		// by the same process that failed the first task execution anyway, which is a very rare case.
 		#endif
-		return IAsyncFileSystem::filesystem()->open(fullPath, flags, 0644);
+		Future<Reference<IAsyncFile>> f = IAsyncFileSystem::filesystem()->open(fullPath, flags, 0644);
+
+		if(g_network->isSimulated()) {
```

Comment:
This seems not the clang-format. Although we haven't made clang-format as a hard requirement, we can start apply clang-format for any lines we change. 
You can run `git clang-format [commit_point_this_PR_is_based_on]` to automatically format the changes using clang-format.